app/models/specification.rb

```diff
@@ -18,6 +30,14 @@ class Specification < ApplicationRecord
 
   include Identifiable
 
+  ###
+  # @description: If there's no specification for the user's company and the selected domain
+  #   to map to, then it's the spine.
+  ###
+  def mark_as_spine
```

Comment:
Why define an extra method when you can simply call `spine!` directly?

Reply:
Well, you're really right. Where's my head at!? 🤪